### ISSUE TITLE
Thank you for sharing the code and having some training problems

### ISSUE THREAD

**Issue by j81-xka** [peisuke/AutoVC.pytorch#3]
hello，Did my training process use all the voice of WAV48 to execute the 600epochs in your code to get the conversion result of your example below?What is the approximate convergence of your reconstruction losses?I used parts of WAV48 without rebuilding the loss and did not get very good results.--src-emb [path to SRC embedding] \ Is the --src-emb [path to SRC embedding] \ used in one or more speeches?Is it invisible?I am trying to convert the embedding with an invisible speech.Thank you for your code, if you have time, I hope to get your reply.
